mkdocs/docs/index.md

```diff
@@ -1,22 +1,156 @@
 # IREE
 
-** * UNDER CONSTRUCTION ** *
+IREE (**I**ntermediate **R**epresentation **E**xecution **E**nvironment[^1]) is
+an [MLIR](https://mlir.llvm.org/)-based end-to-end compiler that lowers Machine
+Learning (ML) models to a unified IR optimized for real-time inference on
+mobile/edge devices against heterogeneous hardware accelerators. IREE also
+provides flexible deployment solutions for its compiled ML models.
 
-## Overview
+## Introduction
 
-<!-- TODO(??): copy introduction from README.md -->
-<!-- TODO(??): describe this website? -->
+### Key features
 
-## Components
+- [x] Support for dynamic shapes, flow control, streaming, and other advanced
+model features
+- [x] Ahead-of-time compilation of scheduling and execution logic together
+- [x] Low overhead, pipelined execution for efficient power and resource usage
+- [x] Binary size as low as 30KB on embedded systems
+- [x] Debugging and profiling support
+- [ ] Support for running quantized models (coming soon!)
 
-### Frontends
+### Support matrix
 
-<!-- TODO(??): explain what frontends are, introduce options -->
+The IREE compiler tools run on :fontawesome-brands-linux: Linux,
+:fontawesome-brands-windows: Windows, and :fontawesome-brands-apple: macOS.
+IREE supports importing from a variety of ML frameworks and running models on a
+wide range of platforms, CPU architectures, and hardware accelerators.
 
-### Backends
+Model formats
 
-<!-- TODO(??): explain what backends are, introduce options -->
+- [x] TensorFlow
```

Comment:
How do we order this? Should we have some rule? (Like supported -> unsupported, and then alphabetically?)

Similarly for the other tables.

Reply:
I loosely ordered first by supported, unsupported, then second by expected # of users